configuration/bahmniforms/second_vital.json

```diff
@@ -1,89 +1,83 @@
+        "dataType": "org.bahmni.customdatatype.datatype.FileSystemStorageDatatype",
+        "uuid": "daeddb2d-832b-4855-81da-a4fcdd1ff698"
       },
       {
-        "locale": "es",
-        "labels": {},
-        "concepts": {
-          "HEIGHT_6": "Altura",
-          "WEIGHT_7": "Peso",
-          "OXYGEN_SATURATION_5": "SaO2",
-          "BLOOD_PRESSURE_2": "Presión sanguínea",
-          "TEMPERATURE_3": "T",
-          "RESPIRATORY_RATE_4": "RR",
-          "DIASTOLIC_10": "Diastólica",
-          "HEAD_CIRCUMFERENCE_8": "Circunferencia de la cabeza",
-          "PULSE_1": "Legumbres",
-          "SYSTOLIC_9": "Sistólica"
-        },
-        "formName": "Second Vitals",
-        "formUuid": "b4f52108-fd2b-4a09-896c-7a5e1c4e7e10",
-        "version": "1",
-        "referenceVersion": null,
-        "referenceFormUuid": null
+        "value": "[{\"display\":\"Second Vitals\",\"locale\":\"en\"},{\"display\":\"Second Vitals\",\"locale\":\"es\"}]",
+        "dataType": "org.bahmni.customdatatype.datatype.FormNameTranslationDatatype",
+        "uuid": "8e069fda-337f-4c08-84d7-3d1739c1d5c8"
       }
     ]
-  }
\ No newline at end of file
+  },
+  "translations": [
+    {
+      "locale": "en",
+      "labels": {},
+      "concepts": {
+        "DIASTOLIC_4": "Diastolic",
+        "SYSTOLIC_3": "Systolic",
+        "BLOOD_PRESSURE_2": "Blood Pressure",
+        "TEMPERATURE_5": "Temperature",
+        "OXYGEN_SATURATION_7": "Oxygen Saturation",
+        "RESPIRATORY_RATE_6": "Respiratory Rate",
+        "PULSE_1": "Pulse"
+      },
+      "formName": "Second Vitals",
+      "formUuid": "b4f52108-fd2b-4a09-896c-7a5e1c4e7e10",
+      "version": "1",
+      "referenceVersion": null,
+      "referenceFormUuid": null
+    },
+    {
+      "locale": "es",
+      "labels": {},
+      "concepts": {
+        "DIASTOLIC_4": "Diastólica",
+        "SYSTOLIC_3": "Sistólica",
+        "BLOOD_PRESSURE_2": "Presión sanguínea",
+        "TEMPERATURE_5": "La temperatura",
+        "OXYGEN_SATURATION_7": "Saturación de oxígeno",
+        "RESPIRATORY_RATE_6": "La frecuencia respiratoria",
+        "PULSE_1": "Legumbres"
```

Comment:
Is this the right translation?
Ot should be more "Pulso" I believe.

Reply:
Done